### PR TITLE
Updated license requirements.md

### PR DESCRIPTION
**Pull request by nibandyo**
Updated license requirements.

### CODE REVIEW

**Comment by nibandyo** [comment]
Updated license requirements

File: OneDrive/limit-access.md [hunk 1]
```diff
@@ -24,19 +24,11 @@ Use this setting if you want to allow only users in specified security groups to
 
 ## Requirements
 
-To enable this feature, you need a subscription to one of the following plans:
+To access and use this feature, you organization must have one of the following subscriptions:
```
Comment (JohanFreelancer9) [inline OneDrive/limit-access.md:27]:
```suggestion
To access and use this feature, your organization must have one of the following subscriptions: 
```

File: OneDrive/limit-access.md [hunk 2]
```diff
@@ -24,19 +24,11 @@ Use this setting if you want to allow only users in specified security groups to
...
-- Microsoft 365 E5
-
-- Microsoft 365 E5 Compliance
-
-- Microsoft 365 E5 Information Protection and Governance
-
-- Microsoft 365 A5
+- Office 365 E5 / A5 
+- Microsoft 365 E5 / A5 
+- Microsoft 365 E5 Compliance / A5 Compliance
```
Comment (JohanFreelancer9) [inline OneDrive/limit-access.md:31]:
```suggestion
- Office 365 E5/A5 
- Microsoft 365 E5/A5 
- Microsoft 365 E5 Compliance/A5 Compliance 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @nibandyo, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@nibandyo, kindly also commit the suggested change to line 27 to correct the typo. Many thanks.

**Comment by nibandyo** [comment]
Change updated

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**